acceptance/tests/propagation/negative/prop_file.rb

```diff
@@ -18,7 +18,7 @@
 verify_content_command = "cat /cygdrive/c/#{parent_name}/#{target_name}"
 file_content_regex = /\A#{file_content}\z/
 
-verify_manifest = /\{ identity => '.*\\bob', rights => \["full"\], affects => 'self_only' \}/
+verify_manifest = /\{ affects => 'self_only', identity => '.*\\bob', rights => \['full'\s+\] \}/
```

Comment:
The test already does that by actually verifying the ACL with `icacls`.

But the test also asserts the explicit Puppet output along the way, which is the only thing fixed here.